src/app/core/admin/admin-dashboard/admin-dashboard.component.html

```diff
@@ -0,0 +1,109 @@
+<div class="header flex-row flex-wrap justify-space-between">
+  <app-view-title
+    class="admin-ui-title"
+    i18n="page title"
+    [displayInPlace]="true"
+  >
+    Edit Dashboard Widgets
+  </app-view-title>
+
+  <div class="flex-row gap-small save-buttons">
+    <button mat-raised-button color="accent" (click)="save()" i18n>Save</button>
+    <button mat-stroked-button (click)="cancel()" i18n>Cancel</button>
+  </div>
+</div>
+
+<div class="hint-banner" i18n>
+  You can customize the widgets on your dashboard to highlight the information
+  that is most useful to you. You can personalize the dashboard to show the key
+  insights you need.
+</div>
+
+<div class="overall-container flex-row" [class.disabled]="isDisabled">
+  <!-- WIDGET PREVIEW AREA -->
+  <div
+    cdkDropList
+    [cdkDropListData]="dashboardConfig.widgets"
+    (cdkDropListDropped)="drop($event)"
+    cdkDropListOrientation="mixed"
```

Comment:
drag&drop of a widget between two rows of widgets suddenly creates a whole empty row --> needs optimization in the drag preview (unfortunately I can't take screenshots while dragging something ... 🙄 )

@sadaf895 , can you connect with @Abhinegi2 and optimize the drag&drop preview stylings together in a pair programming session? (please bring back the discussion if you should find it needs any larger restructuring, though)

Reply:
maybe we can even share/import the styling (e.g. dotted lines as containers) between the drag&drop layout in admin of entity-forms and here for consistency?

Reply:
for drag&drop of form fields, we also let the user drag it from anywhere in the field (although we also show the drag-handle icon). For consistency and easier use, I would suggest we allow the same for dashboard widgets (i.e. let users drag wherever they click on the widget)